include/wil/resource.h

```diff
@@ -3736,6 +3738,7 @@ namespace wil
     template <typename T>
     using unique_hlocal_ptr = wistd::unique_ptr<T, hlocal_deleter>;
 
+#if defined(__PLACEMENT_NEW_INLINE)

```

Comment:
This does not protect against multiple includes. E.g. including `resource.h`, then including `new`, then including `resource.h` again.

Reply:
It's not supposed to. What it does is preventing compilation errors in `/permissive-` mode when `<wil/resource.h>` is included before `<new>`

Reply:
What I'm saying is that it _needs_ to be able to support that.

Reply:
not obvious but wil/resource.h requires being included after the types it supports. and sometimes that means being included multiple times.